OpenMage/magento-lts
# Cleanup: disable fixerio in config

### Description (*)

Update config file to disable fixerio service by default

### Related Pull Requests
---

### Fixed Issues (if relevant)

1. Fixes OpenMage/magento-lts#1094

### Manual testing scenarios (*)

1. Login to Backend -> System -> Configuration -> Currency setup
2. Open Fixer.io Tab
3. Enabled should be "no"

### Contribution checklist (*)
